packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -356,28 +357,32 @@ export default function CommentActionButtons({
           color={ButtonColor.BlueCheese}
         />
       </SimpleTooltip>
-      <AwardButton
-        appendTo={appendTo}
-        type="COMMENT"
-        entity={{
-          id: comment.id,
-          receiver: comment.author,
-          numAwards: comment.numAwards,
-        }}
-        pressed={!!comment.userState?.awarded}
-        post={post}
-      />
-      {!!comment.numAwards && (
-        <Typography
-          className="mr-3"
-          type={TypographyType.Callout}
-          color={TypographyColor.Tertiary}
-          bold
-        >
-          {/* TODO feat/transactions show most expensive award image next to count */}
-          {largeNumberFormat(comment.numAwards)} Award
-          {comment.numAwards > 1 ? 's' : ''}
-        </Typography>
+      {checkCoresRoleNotNone(user) && (
```

Comment:
Missing few checks:
- mobile profile, shows the header for cores
- awards shown on profile

Reply:
Still don't see payment and cores page updated

Reply:
Do you mean /cores and /cores/payment page?

Reply:
If so, it's in the CoresLayout, rather on each page

Reply:
yes

Reply:
ah you moved it to layout